lib/build_local_glossary.rb

```diff
@@ -0,0 +1,12 @@
+require_relative "./google_sheets_service.rb"
+require "active_support/core_ext/object/blank"
+
+glossary_contents = GoogleSheetsService.new.spreadsheet_values
+
+# reject entries with no acronym (entry[2]) or no definition (entry[1])
+glossary_contents = glossary_contents.reject { |entry| entry[1]&.strip.blank? || entry[2]&.strip.blank? }
+
+# arrange the entries as a hash of ACRONYM => Definition pairs
+acronym_defs = glossary_contents.map { |entry| [entry[2].strip, entry[1].strip] }.to_h
```

Comment:
Optional refactoring: You might be able to combine this enumeraton with the open above by using [filter_map](https://ruby-doc.org/core-3.1.2/Enumerable.html#method-i-filter_map).